keyboards/keycapsss/plaid_pad/rules.mk

```
@@ -31,7 +31,7 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
-ENCODER_ENABLE = yes
 
 LAYOUTS = ortho_4x4
-LAYOUTS_HAS_RGB = no
+
+DEFAULT_FOLDER = keycapsss/plaid_pad/rev2
```

This should be
```suggestion
DEFAULT_FOLDER = keycapsss/plaid_pad/rev1
```
to retain compatibility with the current make command.